hca/regenerate_api.py

```diff
@@ -320,39 +359,36 @@ def generate_python_bindings(test_api_path=None):
 
     # Blow away previous files in the autogenerated folder and remake that directory
     dirname = os.path.dirname(__file__)
+    api_function_path = os.path.join(dirname, "api", "__init__.py")
+    if os.path.exists(api_function_path):
+        os.remove(api_function_path)
+        open(api_function_path, 'a').close()
+
     autogenerated_path = os.path.join(dirname, "api", "autogenerated")
     if os.path.exists(autogenerated_path):
         shutil.rmtree(autogenerated_path)
     os.mkdir(autogenerated_path)
     open(os.path.join(autogenerated_path, "__init__.py"), 'a').close()
 
-    for (endpoint_name, endpoint_info) in param_holders.items():
-
-        template_loader = FileSystemLoader(searchpath=os.path.join(dirname, "api", "templates"))
+    function_payload = {'classes': []}
 
-        # An environment provides the data necessary to read and parse our templates. Pass in the loader object here.
-        template_env = Environment(loader=template_loader)
+    for (endpoint_name, endpoint_info) in param_holders.items():
```

Comment:
i think it's just 

`for endpoint_name, endpoint_info in param_holders.items():`